pwiz_tools/Skyline/TestUtil/TestFunctional.cs

```diff
@@ -384,23 +384,51 @@ private static void SkylineInvoke(Action act)
         {
             if (null != SkylineWindow)
             {
-                SkylineWindow.Invoke(act);
+
```

Comment:
I guess I would prefer a function to code duplication. That would keep this block a one-liner. SkyineInvoke(SkylineWindow, act);